static/app/components/replays/replayCurrentUrl.tsx

```diff
@@ -27,6 +29,20 @@ function ReplayCurrentUrl() {
     );
   }
 
+  if (url.includes('[Filtered]')) {
+    return (
+      <Tooltip
```

Comment:
Disabling the input also means tooltips refuse to show over the disabled copy button, so I'm getting around this by adding a tooltip for the entire input box. i.e. hovering over the URL or the copy button shows the tooltip (only when disabled)